.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+env: {}
+
+# DO NOT EDIT BELOW, USE: npx ghat fregante/ghatemplates/webext --set 'on.schedule=[{"cron": "42 17 * * 4"}]'
+
+name: Release
+on:
+  workflow_dispatch: null
+  schedule:
+    - cron: 42 17 * * 4
+jobs:
+  Version:
+    outputs:
+      created: '${{ steps.daily-version.outputs.created }}'
+      version: '${{ steps.daily-version.outputs.version }}'
```

Comment:
`js-yaml` messes with the template’s quotations, I hope that it doesn't affect GitHub

Reply:
Auto-deployment in less than 12 hours. Let's see how it goes.